core/common/src/main/java/alluxio/grpc/GrpcChannelBuilder.java

```diff
@@ -54,16 +46,10 @@ private GrpcChannelBuilder(GrpcServerAddress address, AlluxioConfiguration conf)
     // Read constants.
     mAuthType =
         mConfiguration.getEnum(PropertyKey.SECURITY_AUTHENTICATION_TYPE, AuthType.class);
-    mShutdownTimeoutMs =
-        mConfiguration.getMs(PropertyKey.NETWORK_CONNECTION_SHUTDOWN_TIMEOUT);
-    mHealthCheckTimeoutMs =
-        mConfiguration.getMs(PropertyKey.NETWORK_CONNECTION_HEALTH_CHECK_TIMEOUT);
 
     // Set default overrides for the channel.
     mChannelKey = GrpcChannelKey.create(conf);
     mChannelKey.setServerAddress(address);
-    mChannelKey.setMaxInboundMessageSize(
-        (int) mConfiguration.getBytes(PropertyKey.USER_NETWORK_MAX_INBOUND_MESSAGE_SIZE));
```

Comment:
Is there a reason we're not making this configurable anymore?

Reply:
It is configurable. Just not per channel but per connection. You can set different defaults to connections for streaming vs rpc.